packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -863,9 +866,9 @@ if (missingLangs.length) {
 
 const areDistResources = conf.get('static_directory') === 'dist';
 conf.set('are_dist_resources', areDistResources);
-const options = {
-  strict: true,
-};
+// const options = {
+//   strict: true,
+// };
 
 // validate the configuration based on the above specification
-conf.validate(options);
+// conf.validate(options);
```

Comment:
Validation is broken with the current `scopedKeys.validation` definition using convict 6, something about having urls as keys breaks it. Since the code still works post-validation I'm in favor of skipping validate, for now at least.